oak_runtime/src/lib.rs

```diff
@@ -690,6 +690,19 @@ impl Runtime {
         serialize_label(self.get_node_privilege(node_id).into(), capacity)
     }
 
+    /// Returns the [`Label`] associated with the channel handle.
+    fn get_channel_label(
```

Comment:
Perhaps reuse this function in `get_serialized_channel_label` rather than repeating the same logic?

Reply:
Thanks. Done.